lib/matplotlib/markers.py

```diff
@@ -213,12 +213,19 @@ def __init__(self, marker=None, fillstyle=None):
 
         Parameters
         ----------
-        marker : str or array-like, optional, default: None
+        marker : str, array-like, `~.path.Path`, or `~.markers.MarkerStyle`, \
+        default: None
```

Comment:
I think this needs to be de-indented or we will have a bunch of spaces in the docstring.

Reply:
The rendered doc looks like this:

![image](https://user-images.githubusercontent.com/23121882/76700169-3d31a980-66b5-11ea-96c0-285319bd0421.png)

So I think the indentation is fine?